express/scripts/instrument.js

```diff
@@ -956,73 +956,81 @@ loadScript(martechURL, () => {
   Context.set('segments', []);
 
   function getAudiences() {
-    const visitorId = _satellite.getVisitorId ? _satellite.getVisitorId() : null;
-    const ecid = visitorId ? visitorId.getMarketingCloudVisitorID() : null;
-
-    if (ecid) {
-      w.setAudienceManagerSegments = (json) => {
-        if (json?.segments?.includes(RETURNING_VISITOR_SEGMENT_ID)) {
-          const audiences = Context.get('audiences');
-          const segments = Context.get('segments');
-          audiences.push(ENABLE_PRICING_MODAL_AUDIENCE);
-          segments.push(RETURNING_VISITOR_SEGMENT_ID);
-
-          if (useAlloy) {
-            _satellite.track('event', {
-              xdm: {},
-              data: {
-                eventType: 'web.webinteraction.linkClicks',
-                web: {
-                  webInteraction: {
-                    name: 'pricingModalUserInSegment',
-                    linkClicks: {
-                      value: 1,
+    const getSegments = (ecid) => {
+      if (ecid) {
+        w.setAudienceManagerSegments = (json) => {
+          if (json?.segments?.includes(RETURNING_VISITOR_SEGMENT_ID)) {
```

Comment:
This section is failing because it is not an integer.  I'm not sure how it was working before...

![image](https://user-images.githubusercontent.com/1048648/181367230-63a1d14a-f600-4c92-8c5f-bd264627c9a5.png)

Reply:
I think we just need to make the `const RETURNING_VISITOR_SEGMENT_ID` an int.

Reply:
I went ahead and fixed this.